data/data/alibabacloud/cluster/vpc/outputs.tf

```diff
@@ -23,11 +23,16 @@ output "eip_ip" {
 }
 
 output "slb_ids" {
-  value = [alicloud_slb_load_balancer.slb_external.id, alicloud_slb_load_balancer.slb_internal.id]
+  value = concat(alicloud_slb_load_balancer.slb_external[*].id, [alicloud_slb_load_balancer.slb_internal.id])
+}
+
+output "slb_group_length" {
+  // 1 for private endpoints and 1 for public endpoints
+  value = "2"
```

Comment:
Why is this hard-coded to 2 rather than using the actual length of `slb_ids`?

Reply:
Refer to AWS's usage of aws_lb_target_group_arns_length in cluster/vpc/output.tf.
I updated this part and the test looks ok.